apps/explorer-e2e/src/integration/home.cy.js

```diff
@@ -0,0 +1,141 @@
+context('Home Page', function () {
+  before('visit home page', function () {
+    cy.visit('/');
+  });
+
+  describe('Stats page', function () {
+    const statsValue = '[data-testid="stats-value"]';
+
+    it('Should show connected environment', function () {
+      const deployedEnv = Cypress.env('environment').toUpperCase();
+      cy.get('[data-testid="stats-environment"]').should(
+        'have.text',
+        `/ ${deployedEnv}`
+      );
+    });
+
+    it('should show connected environment stats', function () {
+      const statTitles = [
```

Comment:
suggestion (non-blocking): maybe object instead? as it's long AF and easy to make mistake

Reply:
Done